### PR TITLE
Init dsl demo

### PR DESCRIPTION
**Pull request by banadiga**

### CODE REVIEW

**Review by iverhun — APPROVED** [review]

File: dsl-executor-service/src/main/java/com/lohika/jclub/dsl/service/DslController.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+@RestController
+@RequestMapping(path = "/dsl")
+public class DslController {
```
Comment (flylight) [inline dsl-executor-service/src/main/java/com/lohika/jclub/dsl/service/DslController.java:25]: Why DSL ? We have some specific domain for this script, right? Then lets make this controller represents actual meaning.

File: dsl-executor-service/src/main/java/com/lohika/jclub/dsl/service/DslController.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    MyDsl dsl = new MyDsl(ratingServiceClient, storageServiceClient);
+
+    CompilerConfiguration configuration = new CompilerConfiguration();
+    configuration.setScriptBaseClass(DelegatingScript.class.getName());
+
+    GroovyShell groovy = new GroovyShell(configuration);
+
+    DelegatingScript delegatingScript = (DelegatingScript) groovy.parse(script);
+    delegatingScript.setDelegate(dsl);
+    delegatingScript.run();
```
Comment (flylight) [inline dsl-executor-service/src/main/java/com/lohika/jclub/dsl/service/DslController.java:50]: Here is too much logic as for Controller. Lets create a service layer and move all logic into that class. Lets use controller only as communication layer

File: dsl-executor-service/src/main/java/com/lohika/jclub/dsl/service/DslController.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  private String basepath;
+
+  @Autowired
+  private StorageServiceClient storageServiceClient;
+
+  @Autowired
+  private RatingServiceClient ratingServiceClient;
+
+  @GetMapping(path = "/{scriptName}")
+  public Object runScript(@PathVariable(name = "scriptName") String scriptName) throws IOException {
```
Comment (flylight) [inline dsl-executor-service/src/main/java/com/lohika/jclub/dsl/service/DslController.java:37]: Should this have @ResponseBody annotation or no?

Comment (flylight) [reply dsl-executor-service/src/main/java/com/lohika/jclub/dsl/service/DslController.java:37]: And why Object?

**Review by flylight — COMMENTED** [review]
No tests

**Pull request merged**